docs/hopsml/experiment.rst

```diff
@@ -294,8 +311,23 @@ Experiments service provides a unified view of all the experiments run using the
     :align: center
     :figclass: align-center
 
-Experiments service makes it possible to a TensorBoard that aggregates all the TensorBoard events for a particular run. Since applications can write TensorBoard events to HDFS as they are training this TensorBoard provides a unified view over the whole training process as it is happening over multiple machines. It may also be used to view experiments which have already completed. In the example image below a Parallel Experiment was run using Differential Evolution, as can be seen two hyperparameter combinations were evaluated at the same time (X-axis is wall-clock time).
+In the wrapper function users can return images or files they want to associate with the experiment by returning the local file path to the image. Attached files can then be viewed in the experiments service.
+
+.. _experiment_plot.png: ../_images/experiment_plot.png
+.. figure:: ../imgs/experiment_plot.png
+    :alt: TensorBoard
+    :target: `experiment_plot.png`_
+    :align: center
+    :figclass: align-center
+
+TensorBoard for the whole application is just a click-away. Simply click the TensorFlow icon to start a TensorBoard to visualize the Experiment while it is training or after the experiment is finished. The TensorBoard is opened in a separate tab which means you may need to allow for the popup to appear in your browser.
```

Comment:
Hyperbole.
A TensorBoard can be shown for the experiment by clicking on the TensorFlow icon. TensorBoard can visualize the experiment results (for example, the accuracy of training) both while the experiment is running as well as after the experiment has completed.

separate tab in your browser, which means you may need to allow the popup to appear in your browser (typically you get a notification icon in the searchbar for your browser - click on it to allow the popup).